runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorRequestType.java

```diff
@@ -0,0 +1,57 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * @author Maithem
+ */
+@AllArgsConstructor
+public enum OrchestratorRequestType {
+
+    /**
+     * Query a workflow id
+     */
+    QUERY(0),
+
+    /**
+     * Add a new node to the cluster
+     */
+    ADD_NODE(1),
+
+    /**
+     * Remove node from the cluster
+     */
+    REMOVE_NODE(2);
+
+    @Getter
+    public int type;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Lower the visibility of this field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3066)